src/samples/server/Elsa.Samples.Server.Host/README.md

```diff
@@ -119,6 +119,16 @@ YesSql Postgre
     ...
 ```
 
+### Mandotory features
+
+"Elsa:Features:DefaultPersistence" is a mandotory feature resposible for Elsa Core persistence and must be enabled
+"Elsa:Features:WorkflowSettings" is a mandotory feature resposible for the Workflow Blueprint Settings persistence functionality and must be enabled
```

Comment:
The workflow settings feature should **not** be mandatory. None of the core stuff should have any notion about this.